mflowgen/Tile_PE/construct.py

```diff
@@ -23,22 +23,43 @@ def construct():
   adk_view = 'multicorner-multivt'
   pwr_aware = True
 
+  flatten = 3
+  os_flatten = os.environ.get('FLATTEN')
+  if os_flatten:
+      flatten = os_flatten
+
+  # RTL power estimation
+  rtl_power = False
+  if os.environ.get('RTL_POWER') == 'True':
+      pwr_aware = False
+      rtl_power = True
+
+  # DC power estimation
+  synth_power = False
+  if os.environ.get('SYNTH_POWER') == 'True':
+      synth_power = True
+      pwr_aware = False
+
   parameters = {
     'construct_path'    : __file__,
     'design_name'       : 'Tile_PE',
     'clock_period'      : 4,  # Change to 250 MHz to match Mem.
     'adk'               : adk_name,
     'adk_view'          : adk_view,
     # Synthesis
-    'flatten_effort'    : 3,
+    'flatten_effort'    : flatten,
```

Comment:
We check a number in the genus solution - this could be an issue

Reply:
This holds a number, but defaults to 3.